pom.xml

```diff
@@ -267,6 +273,35 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>3.0.0-M7</version>
+                    <dependencies>
```

Comment:
Since you're cleaning everything up anyway, I believe it makes sense to add just the same configuration for `maven-surefire-plugin`, too.

Reply:
added